src/scripts/start_valheim.sh

```diff
@@ -38,84 +42,92 @@ log "Name: ${NAME}"
 log "World: ${WORLD}"
 log "Public: ${PUBLIC}"
 log "Password: (REDACTED)"
+echo "Loading........."
```

Comment:
Would this make more sense as a `log`?

Reply:
Whoooooooooooooooooooooooooooops this is a debug log i forgot :P